charts/gateway-helm/Chart.yaml

```diff
@@ -1,9 +1,9 @@
 apiVersion: v2
-name: envoy-gateway
+name: gateway-helm
```

Comment:
this can still be `envoy-gateway`, right ?

Reply:
Okay

Reply:
This is me, sorry forks.. Not being hacked...